docs/src/tutorials/tutorial_2.md

```diff
@@ -0,0 +1,125 @@
+
+Possible modifications range in complexity, from simply altering parameter values, to adjusting an existing component, to adding a brand new component. These will take advantage of the public API listed [here](http://anthofflab.berkeley.edu/Mimi.jl/dev/reference/), as well as other functions listed in the Mimi Documentation.
+
+## Parametric Modifications: The API
+
+Several types of changes to models revolve around the parameters themselves, and may include updating the values of parameters and changing parameter connections without altering the elements of the components themselves or changing the general component structure of the model.  The most useful functions of the common API in these cases are likely **`update_param(s)!`, `disconnect_param!`, and `connect_param!`**.  For detail on these functions see the API reference [here](http://anthofflab.berkeley.edu/Mimi.jl/dev/reference/).
+
+When `set_param!` is called in the original model, it creates an external parameter by the name provided, and stores the provided scalar or array value. The functions `update_param!` and `update_params` allow the user to change the value associated with this external parameter.  Note that if the external parameter has a `:time` dimension, use the optional argument `update_timesteps=true` to indicate that the time keys (i.e., year labels) associated with the parameter should be updated in addition to updating the parameter values.
+
+```julia
+update_param!(mymodel, :parametername, newvalues) # update values only 
+
+update_param!(mymodel, :parametername, newvalues, update_timesteps=true) # also update time keys
+```
+
+Also note that in the code above,`newvalues` must be the same size and type (or be able to convert to the type) of the old values stored in that parameter.
+
+If a user wishes to alter the connections within an existing model, `disconnect_param!` and `connect_param` can be used in conjunction with each other to update the connections within the model, although this is more likely to be done as part of larger changes involving components themslves, as discussed in the next subsection.
+
+## Parametric Modifications: DICE Example
+
+### Step 1. Download DICE
+
+The first step in this process is downloading the DICE model.  Open a Julia REPL (here done with the alias command `Julia`) and navigate to the folder where you would like to download DICE.
+```
+Julia 
+cd("directory")
+```
```

Comment:
Add a comment after `cd` command indicating that user must provide real dir name